### PR TITLE
Revert copy clipboard changes from PR #93

### PR DESCRIPTION
**Pull request by lewis-sanchez**
This PR is a part of https://github.com/microsoft/azuredatastudio/issues/25632

This PR reverts #93

### CODE REVIEW

**Comment by caohai** [comment]
Screenshot 2024-08-22 at 14 00 44

It appears that the file indentation format is updated in this PR, causing the diff to be the entire file. I can see the actual change if I toggle "ignore whitespaces", can you fix this?

**Review by caohai — CHANGES_REQUESTED** [review]
Please fix the diff in the PR to only include the actual changes.

File: src/main.ts [hunk 1]
```diff
@@ -588,6 +589,11 @@ export class QueryFeature extends SqlOpsFeature<undefined> {
 		};
 
 		let copyResults = (params: azdata.CopyResultsRequestParams): Thenable<azdata.CopyResultsRequestResult> => {
+			params.copyFromUIProcess = true;
```
Comment (caohai) [inline src/main.ts:592]: I don't think this is the right place to do this check, is there any reason you don't do this in STS?
Plus this code will be running in ADS, so I imagine `os.platform()` will return the OS reported by ADS processes, I believe we want to check the OS that STS is running from, right?

Comment (lewis-sanchez) [reply src/main.ts:592]: So then I just keep the copyFromUIProcess flag, set that in the query editor from ADS. As for the changes in this file (src/main.ts), I get rid of all of this.

**Comment by lewis-sanchez** [comment]
The setting that allows users to have ADS do the copying to the clipboard in case the query provider's copy implementation doesn't work.
image

The ADS PR that added this setting: https://github.com/microsoft/azuredatastudio/pull/23363

**Review by kburtram — APPROVED** [review]

**Pull request merged**